packages/meditrak-server/src/tests/routes/GETAnswers.test.js

```diff
@@ -37,6 +37,8 @@ describe('Permissions checker for GETAnswers', async () => {
   let filterString;
 
   before(async () => {
+    await resetTestData();
```

Comment:
Would it make more sense to reset test data after a batch of tests? And then a single batch can be assumed to have no side effects?

For now I suppose it makes more sense to keep it rather than find all the places that assumption isn't true

Reply:
I tried adding a reset to every batch by using a `beforeEach` in the scaffolding file, but it broke a lot more than it fixed, so yeah...good idea for the future.